packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -226,12 +229,34 @@ export class CartCheckoutPage {
 		await this.page.click( selectors.submitBillingInformationButton );
 	}
 
+	/**
+	 * Selects a saved card payment method.
+	 *
+	 * @param {string} cardHolderName Name of the card holder associated with the payment method.
+	 */
+	async selectSavedCard( cardHolderName: string ): Promise< void > {
+		const selector = this.page.locator( selectors.existingCreditCard( cardHolderName ) ).first();
+
+		await selector.click();
+	}
+
 	/**
 	 * Enter payment details.
 	 *
+	 * Note that this method will always choose to create
+	 * a new card entry even if the intended payment
+	 * details have already been saved.
+	 *
 	 * @param {PaymentDetails} paymentDetails Object implementing the PaymentDetails interface.
 	 */
 	async enterPaymentDetails( paymentDetails: PaymentDetails ): Promise< void > {
+		// Click on the Credit or debit card input in order
+		// to expand the fields.
+		const cardInputLocator = this.page.locator( `span:has-text("Credit or debit card")` );
+		await cardInputLocator.click();
+
+		// Begin filling in the card details from
+		// top to bottom.
```

Comment:
This change is only tangentially related; it is more of a QoL update to ensure this method is able to always fill out the card details when requested.